src/renderer/ui/components/settings/ToggleableOption.js

```diff
@@ -17,7 +18,8 @@ class SettingsCheckbox extends Component {
   render() {
     return (
       <Checkbox
-        label={this.props.label}
+        label={[this.props.label, this.props.hintLabel
+          ? <span id="checkbox-hint" style={{ display: 'block', fontSize: '0.75em', color: 'gray' }}>{this.props.hintLabel}</span> : null]}
```

Comment:
Added the ID for targeting in the test. But I can change it to a class, style with that and use that for targeting in the test :)